src/app/search/directives/search-story.component.css

```diff
@@ -0,0 +1,121 @@
+.story-image {
+  position: relative;
+  float: left;
+  width: 120px;
+  height: 120px;
+  margin-bottom: 5px;
+}
+image-loader {
+  width: 100%;
+  height: 100%;
+  display: block;
+}
+
+.status {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 3px 10px;
+  z-index: 10;
+  color: #fff;
+  background-color: #444;
+  font-weight: 400;
+  font-size: 11px;
+  text-transform: uppercase;
+  line-height: 18px;
+}
+.status.unpublished {
+  background-color: #ee4d3e;
+}
+
+.story-detail {
+  position: relative;
+  float: right;
+  padding: 10px;
+  width: 200px;
+  height: 120px;
+  line-height: 20px;
+}
+.story-title {
+  font-size: 16px;
+  max-height: 40px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.series-title {
+  max-height: 40px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.meta {
+  position: absolute;
+  bottom: 0;
+  width: 180px;
+  line-height: 20px;
+}
+.duration, .modified, .audio-total {
+  width: calc(100%/3);
+}
+.modified {
+  text-align: right;
+}
+.audio-total {
+}
+.audio-total > i {
+  /* TODO: this isn't the correct icon */
```

Comment:
TBH, I wasn't even sure if the icon in the mockup was referring to audio file count.